spec/cohortmanager.spec.js

```diff
@@ -0,0 +1,245 @@
+const CohortManager = require('../src/cohortManager.js')
+const Student = require('../src/student.js')
+
+describe('Cohort Tests', () => {
+  let cohortManager
+
+  const student1 = new Student(
+    1,
+    'Toby',
+    'Carlo',
+    '0xtobycarlo',
+    'tobycarlo.c@gmail.com'
+  )
```

Comment:
i like the idea of declaring the students here instead of inside each it function 👍